test/functional/plugin/lsp/diagnostic_spec.lua

```diff
@@ -1,20 +1,25 @@
 local helpers = require('test.functional.helpers')(after_each)
+local lsp_helpers = require('test.functional.plugin.lsp.helpers')
 
 local clear = helpers.clear
 local exec_lua = helpers.exec_lua
 local eq = helpers.eq
 local neq = require('test.helpers').neq
 
+local create_server_definition = lsp_helpers.create_server_definition
+
 describe('vim.lsp.diagnostic', function()
   local fake_uri
 
   before_each(function()
-    clear {env={
-      NVIM_LUA_NOTRACK="1";
-      VIMRUNTIME=os.getenv"VIMRUNTIME";
-    }}
+    clear({
+      env = {
+        NVIM_LUA_NOTRACK = '1',
+        VIMRUNTIME = os.getenv('VIMRUNTIME'),
+      },
+    })
 
-    exec_lua [[
+    exec_lua([[
```

Comment:
Would be nice if you could revert the unrelated changes. Makes it easier to review and avoids conflicts on rebase/merge

(We should probably enable lualint & luaformat for the test files here at some point)

Reply:
Sorry about that. I was struggling to get formatting working for the non-test files, so I set up neovim to format with `stylua` on write.

It does make the PR harder to review. I'll revert the unrelated changes here.